src/semantic_version.cr

```diff
@@ -47,21 +71,31 @@ class SemanticVersion
     end
   end
 
+  # :nodoc:
```

Comment:
Why `:nodoc:`? The `<=>` comparison operator should be documented so users know it can be used here.
```suggestion
  # The comparison operator.
  #
  # ```
  # require "semantic_version"  
  # 
  # semver1 = SemanticVersion.new(1, 0, 0)
  # semver2 = SemanticVersion.new(2, 0, 0)
  # semver1 <=> semver2 # => -1
  # ```
```

Reply:
Was thinking since its required by the `Comparable` that the user would look [there](file:///home/george/dev/git/crystal/docs/Comparable.html#%28other%3AT%29-instance-method).

But probably wouldn't hurt to add here as well.